narayana-release-process.sh

```diff
@@ -204,23 +199,27 @@ then
   exit
 fi
 git archive -o ../../narayana-full-$CURRENT-src.zip $CURRENT
-ant -f build-release-pkgs.xml -Drsync.host=${RSYNC_HOST} -Drsync.enabled=${RSYNC_ENABLED} -Dawestruct.executable="awestruct" all
-if [[ $? != 0 ]]
-then
-  echo 1>&2 COULD NOT BUILD Narayana RELEASE PKGS
-  exit
-fi
-cd -
+# build-release-pkgs.xml needs to be updated, update the website manually
+echo " narayana.io needs updating, please update the narayana.io repository (javadoc to unzip into api, project and product documentation and then upload to narayana.io"
 
-# Building and pushing the lra coordinator docker image
-cd ~/tmp/narayana/$CURRENT/sources/jboss-dockerfiles/lra/lra-coordinator
-git checkout $CURRENT
-if [[ $? != 0 ]]
-then
-  echo 1>&2 jboss-dockerfiles: Tag $CURRENT did not exist
-  exit
-fi
+  read -p "Please update narayana.io before continuing. Continue? y/n " NARAYANAIO
+  if [[ $NARAYANAIO == n* ]]
+  then
+    exit
+  fi
+#ant -f build-release-pkgs.xml -Drsync.host=${RSYNC_HOST} -Drsync.enabled=${RSYNC_ENABLED} -Dawestruct.executable="awestruct" all
```

Comment:
Let's delete this commented out block about build-release-pkgs.xml - is that OK with you?